### ISSUE TITLE
Add Option for logless vpn

### ISSUE THREAD

**Issue by AskMeWhyMyDickIsHard** [Angristan/OpenVPN-install#143]
As I understand it would be as simple as 
echoing 
log /dev/null
status /dev/null

to:
/etc/openvpn/server.conf

Please let me know if I am being completely retarded. 

If you want ill do a pull request and do it myself

**Comment by SupaBly** [comment]
Yes, you are retarded. Logless VPN are not possible with NAT, its here that VPN provider lie.
